src/node_report.cc

```diff
@@ -471,7 +479,24 @@ static void PrintJavaScriptStack(JSONWriter* writer,
     }
     writer->json_arrayend();
   }
-  writer->json_objectend();
+  writer->json_objectstart("errorProperties");
+  Local<Array> props;
+  if (!error.IsEmpty() &&
+      error->GetOwnPropertyNames(context).ToLocal(&props)) {
+    const int length = props->Length();
+    for (int i = 0; i < length; i++) {
+      Local<Value> key = props->Get(context, i).ToLocalChecked();
+      Local<Value> value = error->Get(context, key).ToLocalChecked();
```

Comment:
Ditto here, what if there are exceptions from these calls? `.ToLocalChecked()` is definitely not okay here, but it goes further than that; the report code can be triggered by uncaught exceptions, and I’m pretty sure we wouldn’t want to cause another uncaught exception here.

If there is no better solution, this entire function and/or each loop iteration here should probably be wrapped in a `TryCatch`, to swallow these exceptions silently…